frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -80,9 +80,9 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     templateValidations,
   } = props;
   const asId = prefixedID.bind(null, 'disk');
-  const disk = props.disk || DiskWrapper.EMPTY;
-  const volume = props.volume || VolumeWrapper.EMPTY;
-  const dataVolume = props.dataVolume || DataVolumeWrapper.EMPTY;
+  const disk = props.disk || new DiskWrapper();
```

Comment:
This (wrapper without a resource) looks better than having explicit "empty" objects.